docs/generate_checks.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2021 Security Scorecard Authors
```

Comment:
Nit: maybe change name to generate_checks_readme/generate_checks_md or something along those lines to clarify what this script does.